frontend/apps/reader/modules/readerdictionary.lua

```diff
@@ -849,6 +849,22 @@ function ReaderDictionary:stardictLookup(word, dict_names, fuzzy_search, box, li
         return
     end
 
+    -- If the user disabled all the dictionaries, go away.
+    if dict_names and #dict_names == 0 then
+        -- Dummy result
+        local nope = {
+            {
+                dict = _("N/A"),
+                word = word,
+                definition = _("There are no enabled dictionaries.\nCheck the 'Dictionary settings' menu."),
```

Comment:
It's still English. :-P
```suggestion
                definition = _("There are no enabled dictionaries.\nPlease check the 'Dictionary settings' menu."),
```